BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -751,7 +751,17 @@ void clear(FaceGraph& g)
   CGAL_postcondition(num_faces(g) == 0);
 }
 
-
+/**
+ * \ingroup PkgBGLHelperFct
+ *
+ * returns the degree of the face with halfedge `hd`
+ *
+ * @tparam FaceGraph model of `MutableHalfedgeGraph` and `MutableFaceGraph`
```

Comment:
1. Why mutable?
2. `FaceGraph` already refines `HalfedgeGraph`. No need to mention both: you can just mention `FaceGraph`.